presto-docs/src/main/sphinx/develop/presto-native.rst

```diff
@@ -159,3 +159,26 @@ Async Data Cache and Prefetching
 
     Size of the SSD cache when async data cache is enabled.  Must be zero if
     ``async-data-cache-enabled`` is ``false``.
+
+``enable-old-task-cleanup``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type** ``bool``
+    * **Default value:** ``true``
+    * **Presto on Spark default value:** ``false``
+
+    Enable periodic clean up of old tasks. This is ``true`` for Prestissimo,
+    however for Presto on Spark this defaults to ``false`` as zombie/stuck tasks
+    are handled by spark via speculative execution.
+
+``old-task-cleanup-ms``
```

Comment:
I believe other similar properties to not include units in the name and allow different units, e.g.

old-task-cleanup with values 60ms, 1s, etc.

It would be nice to follow-up with a change to make this consistent.

CC: @amitkdutta